recipes/jwt-cpp/all/patches/0005-fix-picojson-header-location-for-conan.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/include/jwt-cpp/jwt.h b/include/jwt-cpp/jwt.h
+--- a/include/jwt-cpp/jwt.h
++++ b/include/jwt-cpp/jwt.h
+@@ -8,1 +8,1 @@
+-#include "picojson/picojson.h"
++#include "picojson.h"
```

Comment:
should we fix `cpp_info.includedirs` in `picojson`?

Reply:
includedir of conan seems correct: https://github.com/kazuho/picojson/blob/111c9be5188f7350c2eac9ddaedd8cca3d7bf394/Makefile

Reply:
Conan is correct 👍